### ISSUE TITLE
Typo i Local config causes the app to crash when starting up next time.

### ISSUE THREAD

**Issue by haavamoa** [Illedan/OpenRepo#2]
When I had a typo in my `Local` config, the app crashes the next time I start it up.
This makes sense, but I do now know how to get to the config to edit it.

Could add the link to the config in the error message.

```
> openrepo.exe                                                           
Loading...                                                                                                              
Check changes crashed: Could not find a part of the path '<my-broken-path>'    
```

**Comment by Illedan** [comment]
Thanks for reporting. 
I will look into it!

**Comment by Illedan** [comment]
Should be fixed with #4 . Please try again once 0.0.8 is indexed at Nuget.org.
